iaso/api/mobile/org_units.py

```diff
@@ -30,7 +32,34 @@ def get_page_number(self, request):
         return int(request.query_params.get(self.page_query_param, 1))
 
 
-class MobileOrgUnitSerializer(ModelSerializer):
+class ReferenceInstancesFilter(django_filters.rest_framework.FilterSet):
+    last_sync = django_filters.IsoDateTimeFilter(field_name="updated_at", lookup_expr="gte")
+
```

Comment:
BTW, I'm a bit confused by [`version_count`](https://github.com/BLSQ/iaso/blob/039b8066c1a7ffbc41705e2870d22b645d931d8c/docs/pages/dev/reference/API/org_unit_registry.md?plain=1#L358) because we are manipulating `Instance`s.
I'm not sure to understand, let's re-talk about this.

Reply:
As discussed, this was in case we kept the history of all the `Instance`s that once were an accepted reference instance.
Since this is not the case, this `version_count` parameter can be omitted for now. We'll reintroduce it when, and if, it is needed and we'll have the history.